packages/web-components/src/components/cta-section/cta-section.ts

```diff
@@ -108,7 +104,7 @@ class DDSCTASection extends StableSelectorMixin(DDSContentItem) {
 
     childItems?.forEach((elem, index) => {
       /* eslint-disable-next-line prefer-destructuring */
-      newArray[index] = elem.children[1];
+      newArray[index] = (elem as any).children[1];
```

Comment:
Wondering what `.children[1]` is meant for...? BTW TypeScript did a good catch here because `.assignedNodes()` returns any types of nodes but text node, etc. does not have `.children`.

Reply:
`.children[1]` returns a `dds-cta-section-item-copy`, which is where we get the `clientHeight` values that we need.

Reply:
Thanks @IgnacioBecerra for explaining! In such case, we should define `selectorCopy` static property and do something like below so future change in DOM structure won't break the code:

```javascript
const { selectorCopy } = this.constructor as typeof DDSCTASection;
const copyNodes = slotNode?.assignedNodes().reduce((acc, node) => {
  if (node.nodeType === Node.ELEMENT_NODE) {
    acc.push(...(node as Element).querySelectorAll(selectorCopy));
  }
  return acc;
}, []);
```

Search for `selectorSomething` static properties in other components to see the details.

Reply:
That is much cleaner, I like it, thanks @asudoh!